### DISCUSSION TITLE
Certain funbox settings which don't change the word gen should not have PB's/leaderboards disabled

### DISCUSSION THREAD

**Discussion by jmcwilliams403** [monkeytypegame/monkeytype#1566] [Ideas] [answered]
I have ADHD and I've found I performed at my best with Plus One or Plus Two modes enabled because it keeps me from looking at the entire block of text when I don't need to. I found out that funbox settings disable PB's after beating my current best with Plus One enabled, and I felt cheated.

I understand if some of the other funbox settings which change the word gen should disqualify you from the leaderboards, but the ones which don't would theoretically not be a problem.

**Comment by unrealapex** [comment]
> I have ADHD and I've found I performed at my best with Plus One or Plus Two modes enabled because it keeps me from looking at the entire block of text when I don't need to. 

I too have ADHD, what helps me focus is turning off `timer display` so I don't need to ponder and look at how many seconds I have until a test ends while I'm typing. This is more of an accessibility issue. I think Monkeytype should provide an indistinguishable experience for all users, regardless if they have an impairment.

**Comment by Miodec** [comment, marked as answer]
Allowed leaderboards and pbs with `plus one` and `plus two`.

**Comment by ghost** [comment]
Other funbox settings that do not change the word gen such as 'read ahead' also should not have leaderboards/PB's disabled.

**Comment by plainprince** [comment]
Weakspot does kind of change word generation but I think it should still be allowed in leaderboards/PB's.  Plus three should also be allowed in leaderboards/PB's.
